example-ideal.txt

```diff
@@ -0,0 +1,20 @@
+// Implementation example for empty picobot maze
+
+state leftTraverse {
+	move left;
```

Comment:
Yeah, it's a little weird. Usually everything requires a semicolon or nothing does.